TITLE:
RuntimeError: threads can only be started once

LABEL:
bug

STATE:
closed

BODY:
wandb, version 0.8.12
Python 3.7.4
Linux


### Description
Code to reproduce:

```
from fastai.vision import *
import wandb
from wandb.fastai import WandbCallback

wandb.init(project="projectname")

learn = cnn_learner(data, model, metrics=[accuracy, mean_squared_error], callback_fns=[WandbCallback])
learn.model = nn.DataParallel(learn.model)
learn.fit_one_cycle(6)
learn.unfreeze()
learn.fit_one_cycle(40, slice(1e-5,3e-4))
learn.validate()
```

yields: `RuntimeError: threads can only be started once`

